src/api/public/apidocs-new/components/responses/person/not_found.yaml

```diff
@@ -0,0 +1,11 @@
+description: |
+  Not found.
+
+  XML Schema used for body validation: [status.xsd](../schema/status.xsd)
```

Comment:
```suggestion
  XML Schema used for body validation: [status.xsd](../../schema/status.xsd)
```

Reply:
Ouch, thanks!

Reply:
Talked offline and we agreed on disregarding this suggestion.